glyphs-reader/src/font.rs

```diff
@@ -2703,10 +2708,15 @@ impl Instance {
     /// Get the optional postscript name to use for the `fvar` named instance.
     pub fn postscript_name(&self) -> Option<&str> {
         // https://handbook.glyphsapp.com/custom-parameter-descriptions/
-        self.properties
+        // https://github.com/googlefonts/glyphsLib/blob/bd8276222afad582c030f839d90271bc0ee70d0b/Lib/glyphsLib/builder/instances.py#L118
+        ["variablePostscriptFontName", "postscriptFontName"]
```

Comment:
Is there any concern about precedence?

Reply:
> Is there any concern about precedence?

in what sense?

Reply:
this will stop at the first thing it finds, so we will always prefer the first key to the second one.

Reply:
Yes, this was the intention.